packages/web/src/components/collection/desktop/CollectionHeader.tsx

```diff
@@ -225,57 +225,57 @@ export const CollectionHeader = (props: CollectionHeaderProps) => {
               })}
               onClick={isOwner ? handleClickEditTitle : undefined}
             >
-              <Text
-                variant='heading'
-                size='xl'
-                className={cn(styles.titleHeader, fadeIn)}
-                textAlign='left'
-              >
-                {title}
-              </Text>
-              <ClientOnly>
-                {isOwner ? (
-                  <IconPencil className={styles.editIcon} color='subdued' />
-                ) : null}
-              </ClientOnly>
               {isLoading ? (
-                <Skeleton css={{ position: 'absolute', top: 0 }} />
-              ) : null}
+                <Skeleton height='48px' width='300px' />
```

Comment:
seems like it might be better to just have one whole `CollectionSkeleton` component in a separate file, and have one big ternary at the top level that switches between that and the real content, rather than interspersing small skeletons throughout the whole file. but if this pattern was pre-existing, then 🤷

Reply:
yeah I was wondering the same. This was sort of easier cause it ensures parity with the layout, and I've seen them drift before, but I'm open to either